### PR TITLE
KYC-96 Email tests

### PR DESCRIPTION
**Pull request by jchannon**

### CODE REVIEW

File: tests/Apollo.Tests.Unit/Features/EmailAddressTests.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+// See the LICENSE file in the project root for more information.
+
+namespace Apollo.Tests.Unit.Features
+{
+    using System;
+    using Xunit;
+
+    public class EmailAddressTests
+    {
+        [Theory]
```
Comment (JoeStead) [inline tests/Apollo.Tests.Unit/Features/EmailAddressTests.cs]: I think we want a test which proves the validity too

Comment (jchannon) [reply tests/Apollo.Tests.Unit/Features/EmailAddressTests.cs]: done

File: tests/Apollo.Tests.Unit/Features/EmailAddressTests.cs [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        [InlineData("no-at-symbol.com")]
+        public void Should_throw_on_invalid_emailaddress(string emailAddress)
+        {
+            Assert.Throws<ArgumentException>(() => new EmailAddress(emailAddress));
+        }
+
+        [Theory]
+        [InlineData("")]
+        [InlineData(null)]
+        public void Should_throw_on_empty_null_emailaddress(string emailAddress)
```
Comment (JoeStead) [inline tests/Apollo.Tests.Unit/Features/EmailAddressTests.cs]: This is the same test as the one above

Comment (jchannon) [reply tests/Apollo.Tests.Unit/Features/EmailAddressTests.cs]: nope read harder :)

Comment (JoeStead) [reply tests/Apollo.Tests.Unit/Features/EmailAddressTests.cs]: Yeah, I did. It's the same test with different data

Comment (jchannon) [reply tests/Apollo.Tests.Unit/Features/EmailAddressTests.cs]: different exceptions

Comment (JoeStead) [reply tests/Apollo.Tests.Unit/Features/EmailAddressTests.cs]: Shhh

**Pull request merged**